package.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
+  <name>buoy_msgs</name>
+  <version>0.0.0</version>
+  <description>Power Buoy messages (telemetry) and services (commands)</description>
+  <maintainer email="anderson@mbari.org">Michael Anderson</maintainer>
+  <license>TODO: License declaration</license>
+
+  <buildtool_depend>ament_cmake</buildtool_depend>
+  
+  <build_depend>rosidl_default_generators</build_depend>
+  <exec_depend>rosidl_default_runtime</exec_depend>
+  <member_of_group>rosidl_interface_packages</member_of_group>
+  
+  <depend>std_msgs</depend>
```

Comment:
And this one could be an `exec_depend`, see [example](https://github.com/ros2/common_interfaces/blob/969a56b0b59f7f70b5c4561185ea34aaa2b74e1d/geometry_msgs/package.xml#L20).

```suggestion
  <exec_depend>std_msgs</exec_depend>
```